extensions/gdx-jnigen/src/com/badlogic/gdx/jnigen/parsing/JavaMethodParser.java

```diff
@@ -118,6 +118,19 @@ public String getArrayCType () {
 			if (this == DoubleArray) return "double*";
 			throw new RuntimeException("Unknown Array type " + this);
 		}
+		
+		public String getArrayJniType () {

```

Comment:
There is `getArrayCType()`, which is very similar, but the `jfoo`-types are there for a reason?